docs/rust-ci-cd-integration.md

```
@@ -0,0 +1,552 @@
+./bin/goclean scan testdata/rust-samples/sample-project --format json > rust-scan-results.json
+
+# Verify Rust-specific violations were detected
+if grep -q "RUST_" rust-scan-results.json; then
+  echo "✓ Rust-specific violations detected successfully"
+else
+  echo "⚠ No Rust-specific violations detected"
+fi
+```
+
+### Expected Rust Violations
+The CI should check for these violation types:
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)